securedrop/source_app/decorators.py

```diff
@@ -18,7 +18,7 @@ def ignore_static(f):
     a static resource."""
     @wraps(f)
     def decorated_function(*args, **kwargs):
-        if request.path.startswith('/static') or request.path == '/org-logo':
```

Comment:
why remove this `/org-logo` logic?

Reply:
Actually... removing that logic throughout the Source UI, has been an ongoing desire of mine for some time. 

There's no intuitive destination imho for that logo to point to; and that functionality was put in before "Cancel" buttons existed on all the pages as 'the way out.' Likewise, its `<alt>` tag should be changed to simply say "Logo" and not "SecureDrop." (...as the latter is _very_ confusing, especially when it's all that displays as Tor slowly loads the org logo)

Reply:
sorry nina this is just logic regarding how URLs are routed through the application, not actually removing the logo logic in its entirety 😇 

but OK I see why this is being done in 95c0c31fdc38eafc60932d7eec1e6ab57c290025, resolving this convo

Reply:
*blushes* roger that! 😸